packages/inertia-vue/index.d.ts

```diff
@@ -50,3 +50,7 @@ type InertiaLink = FunctionalComponentOptions<InertiaLinkProps>
 export const InertiaLink: InertiaLink
 
 export const InertiaApp: App
+
+export const App: App
+
+export const plugin: PluginObject
```

Comment:
This is this `PluginObject`, and not just `plugin`?

Reply:
It's this one
https://github.com/vuejs/vue/blob/bb253db0b3e17124b6d1fe93fbf2db35470a1347/types/plugin.d.ts#L5

Reply:
Ahh, gotcha. 👍